src/bin/bat/assets.rs

```diff
@@ -18,10 +20,15 @@ pub fn cache_dir() -> Cow<'static, str> {
     PROJECT_DIRS.cache_dir().to_string_lossy()
 }
 
-pub fn clear_assets() {
-    clear_asset("themes.bin", "theme set cache");
-    clear_asset("syntaxes.bin", "syntax set cache");
-    clear_asset("metadata.yaml", "metadata file");
+pub fn clear_assets(matches: &clap::ArgMatches) {
+    let target_dir = matches
+        .get_one::<String>("target")
+        .map(Path::new)
+        .unwrap_or_else(|| PROJECT_DIRS.cache_dir());
```

Comment:
Thanks for working on this 👍 

Would be great if we could take it one step further and
1. Remove all uses of `PROJECT_DIRS.cache_dir()`
2. Calculate its value centrally and pass it down to whoever needs it

That way we don't need to duplicate this code from `fn build_assets()`, and we reduce the use of globals.

And long-term (or in this PR if you wish) we should do the same for for `PROJECT_DIRS.config_dir()`.